README.md

```diff
@@ -1,2 +1,190 @@
-# seeder
-Kiwix Catalog Bittorrent Seeder Companion
+# Kiwix Seeder
+
+`kiwix-seeder` is a simple tool that allows one to manage a Bittorrent seeder for Kiwix Catalog's ZIMs effortlessly.
+
+[![CodeFactor](https://www.codefactor.io/repository/github/kiwix/seeder/badge)](https://www.codefactor.io/repository/github/kiwix/seeder)
+[![License: GPL v3](https://img.shields.io/badge/License-GPLv3-blue.svg)](https://www.gnu.org/licenses/gpl-3.0)
+[![codecov](https://codecov.io/gh/kiwix/seeder/branch/main/graph/badge.svg)](https://codecov.io/gh/kiwix/seeder)
+![PyPI - Python Version](https://img.shields.io/badge/python-3.12+-blue)
+[![Docker](https://ghcr-badge.egpl.dev/kiwix/bittorrent-seeder/latest_tag?label=docker)](https://ghcr.io/kiwix/bittorrent-seeder/)
+
+
+It  consists of a script that runs periodically and which consists mostly of:
+
+- Downloading the Kiwix OPDS Catalog
+- Matching its entries with your defined filters
+- Communicates with your qBittorrent instance (via HTTP)
+  - Removes out-of-date (dropped from Catalog) ZIMs from qBittorrent
+  - Adds new matching ZIM to qBittorrent
```

Comment:
`(which in turns will download them to the filesystem)`

Reply:
You already reported that 😉